authorize/grpc.go

```diff
@@ -89,6 +89,9 @@ func (a *Authorize) Check(ctx context.Context, in *envoy_service_auth_v2.CheckRe
 	evt = evt.Str("email", reply.GetEmail())
 	evt = evt.Strs("groups", reply.GetGroups())
 	evt = evt.Str("session", string(sess))
```

Comment:
i think this session can be nil if during the refresh there was an error (see line 51)